<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>51</width>
      <height>271</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Таблица переходов</string>
    </property>
    <property name="whatsThis">
     <string>Таблица переходов</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid  rgb(217, 217, 217);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/pref/psf/Home/Desktop/Иконки/kedit_7369.png</normaloff>:/pref/psf/Home/Desktop/Иконки/kedit_7369.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>48</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OPButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Оперативная память</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid  rgb(217, 217, 217);</string>
    </property>
    <property name="text">
     <string>ОП</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>16</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>16</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>16</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 181, 222);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Исх. сост.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 181, 222);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>След. сост.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 181, 222);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Условия перехода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 181, 222);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Управляющие сигналы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>811</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border-radius: 10px;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>451</height>
      </rect>
     </property>
     <zorder>gridLayoutWidget</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>OPTable</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>OPTable</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>OPTable</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>OPTable</zorder>
    </widget>
   </widget>
   <widget class="QTableWidget" name="OPTable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>529</width>
      <height>349</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {border: 2px solid  rgb(87, 161, 217);
border-radius: 10px;}</string>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>A</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>B</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>C</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>D</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>E</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>F</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>B</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>C</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string notr="true">00</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>59</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="OPHLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string> 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> 3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>B</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>D</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>E</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>F</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>80</y>
      <width>31</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="OPVLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>B</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>D</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>E</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>F</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="OPTableDecor">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>30</y>
      <width>591</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border: 1px solid  rgb(79, 91, 141);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>OPTableDecor</zorder>
   <zorder>line1</zorder>
   <zorder>OPButton</zorder>
   <zorder>editButton</zorder>
   <zorder>label1</zorder>
   <zorder>label2</zorder>
   <zorder>label3</zorder>
   <zorder>label4</zorder>
   <zorder>OPTable</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>line2</zorder>
   <zorder>line3</zorder>
   <zorder>line4</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>29</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
	background-image: url(:/pref/psf/Home/Desktop/Иконки/Снимок экрана 2016-07-18 в 13.42.36.png);}
QToolButton { background-color: rgba(0,0,0,0); }
QToolButton:hover { border-radius: 6px; border: 1px solid  rgb(150, 150, 150);  }</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>//psf/Home/Desktop/Иконки/misc_6569.png</normaloff>//psf/Home/Desktop/Иконки/misc_6569.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Новый Файл</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/pref/psf/Home/Desktop/Иконки/folder_open.png</normaloff>:/pref/psf/Home/Desktop/Иконки/folder_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Открыть Файл</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/pref/psf/Home/Desktop/Иконки/3floppy_unmount_8962.png</normaloff>:/pref/psf/Home/Desktop/Иконки/3floppy_unmount_8962.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
